### PR TITLE
fix eventrouter issue: ImagePullBackOff

### PR DESCRIPTION
**Pull request by QiaolingTang**
The post action to sync images has been changed, when execute automation testing, always hit issue `ImagePullBackOff`, so I changed the image when deploying eventrouter. @anpingli  PTAL.

### CODE REVIEW

**Comment by QiaolingTang** [comment]
Runner-v3 log: https://url.corp.redhat.com/4127a54

File: features/step_definitions/logging.rb [hunk 1]
```diff
@@ -348,9 +348,23 @@
   step %Q/admin ensures "eventrouter" service_account is deleted from the "openshift-logging" project after scenario/
   step %Q/admin ensures "eventrouter" config_map is deleted from the "openshift-logging" project after scenario/
   step %Q/admin ensures "eventrouter" deployment is deleted from the "openshift-logging" project after scenario/
-  @result = admin.cli_exec(:new_app, file: "#{ENV['BUSHSLICER_HOME']}/testdata/logging/eventrouter/internal_eventrouter.yaml")
-  unless @result[:success]
-    raise "Unable to deploy eventrouter"
+  # get image registry from CLO
+  image_registry = deployment('cluster-logging-operator').container_spec(name: 'cluster-logging-operator').image
+  image_version = cluster_version('version').channel.split('-')[1]
+  #image_version = subscription('cluster-logging').current_csv.split('.', 2)[1]
```
Comment (pruan-rht) [inline features/step_definitions/logging.rb:354]: @QiaolingTang pls remove comment is not used anymore

Comment (QiaolingTang) [reply features/step_definitions/logging.rb]: Done, thanks a lot.

**Pull request merged**